### DISCUSSION TITLE
Is there any way to make fragmented slide, but without list?

### DISCUSSION THREAD

**Discussion by rodion-m** [marp-team/marp#77] [Q&A] [answered]
Hello! I want to make fragmented slide (steps), but without list symbols.
For example:
```
First line
```
{click}
```
Fist line
Next line <--- this is just appeared
```

**Comment by yhatt** [comment, marked as answer]
These may help you:

- https://github.com/marp-team/marpit/issues/195
- https://github.com/marp-team/marpit/issues/209

Marp (Marpit) has [limited fragment support to the list Markdown syntax](https://deploy-preview-280--marpit.netlify.app/fragmented-list).

You should use [a trick by `data-marpit-fragment` HTML data attribute](https://github.com/marp-team/marpit/issues/195#issuecomment-534819826). Or you just can hide bullets from list through [`<style>` (or `<style scoped>`)](https://deploy-preview-280--marpit.netlify.app/theme-css?id=tweak-style-through-markdown) as @vicnala says in https://github.com/marp-team/marpit/issues/209#issuecomment-615257891.

**Reply by rodion-m** [reply]
Thank you very much for a fast response! Can you also tell me where I have to enable `markdown.marp.enableHtml` option?

**Reply by yhatt** [reply]
[`markdown.marp.enableHtml` is a key of `settings.json` for VS Code settings](https://code.visualstudio.com/docs/getstarted/settings). **"Markdown > Marp: Enable HTML"** in the setting editor means exactly same.

![](https://user-images.githubusercontent.com/3993388/55605718-bd1a7880-57b0-11e9-948b-a2a94ea3bc77.gif)

**Reply by rodion-m** [reply]
It works! Thank you very much! And thanks a lot for a great product.
